<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="200557" started-at="2017-12-13T17:00:29Z" finished-at="2017-12-13T17:03:50Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="200557" started-at="2017-12-13T17:00:29Z" finished-at="2017-12-13T17:03:50Z">
      <class name="testing.A">
        <test-method status="PASS" signature="before()[pri:0, instance:testing.A@58ceff1]" name="before" is-config="true" duration-ms="185430" started-at="2017-12-13T22:30:29Z" finished-at="2017-12-13T22:33:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- before -->
        <test-method status="PASS" signature="abc()[pri:0, instance:testing.A@58ceff1]" name="abc" duration-ms="14958" started-at="2017-12-13T22:33:35Z" finished-at="2017-12-13T22:33:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- abc -->
        <test-method status="PASS" signature="after()[pri:0, instance:testing.A@58ceff1]" name="after" is-config="true" duration-ms="17" started-at="2017-12-13T22:33:50Z" finished-at="2017-12-13T22:33:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- after -->
      </class> <!-- testing.A -->
      <class name="testing.B">
        <test-method status="FAIL" signature="abc()[pri:0, instance:testing.B@7c30a502]" name="abc" duration-ms="2" started-at="2017-12-13T22:33:50Z" finished-at="2017-12-13T22:33:50Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at testing.B.abc(B.java:9)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:112)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:176)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- abc -->
      </class> <!-- testing.B -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
